cloud/shared/bin/run

```diff
@@ -13,7 +13,33 @@
 # activate the venv. The activation has to happen in the parent process before the python code is run.
 # This is why we need a bash script that preceeds the python code
 source cloud/shared/bin/python_env_setup
-# TODO(#4612)Download the newest version of the package rather than a fixed version.
+
+# Get the arguments that we want to pass to run.py
+while getopts s:c:t: flag; do
+  case "${flag}" in
+    # The civiform_config file that contains the values to configure the deployment
+    s) sourceconfig=${OPTARG} ;;
+    # The command that the run.py script should execute
+    c) command=${OPTARG} ;;
+    # The tag of the image that should be used for this deployment (e.g. "latest")
+    t) tag=${OPTARG} ;;
+  esac
+done
+
+# If tag is "latest" get the latest release number (eg. v1.26.0)
+if [ $tag == "latest" ]; then
+  tag=$(curl -s "https://api.github.com/repos/civiform/civiform/releases/latest" \
+    | python3 -c "import sys, json; print(json.load(sys.stdin)['tag_name'])")
+fi
+
+# Get the commit sha for the commit at the tip of the CiviForm version being deployed
+tag_url=$(curl -s "https://api.github.com/repos/civiform/civiform/git/refs/tags/${tag}" \
+  | python3 -c "import sys, json; print(json.load(sys.stdin)['object']['url'])")
+commit_sha=$(curl -s "${tag_url}" \
+  | python3 -c "import sys, json; print(json.load(sys.stdin)['object']['sha'])")
```

Comment:
**suggestion (blocking):** please factor out a function for this repeated pattern of piping the output of `curl` through python's JSON lib. E.g.

```shell
#######################################
# Get the value in a JSON document at the given URL.
# Arguments:
#   1: URL of the JSON document
#   2: Path of the value in bracket notation e.g. "['object']['sha']"
# Returns:
#   Value found at the specified path in the JSON document
#######################################
function fetch_json_val() {
  curl -s "${1}" \
    python3 -c "import sys, json; print(json.load(sys.stdin)${2})"
}
```

Reply:
done. thanks for the suggested code!